client/src/components/common/notifications/notifications.jsx

```diff
@@ -18,20 +18,21 @@ const Notifications = ({ user, onPostApply }) => {
       return undefined;
     }
     const { id } = user;
-    socket.emit('createRoom', id);
-    socket.on('like', () => {
-      NotificationManager.info('Your post was liked!');
+    socket.emit(SocketEvent.CREATE_ROOM, id);
+    socket.on(SocketEvent.LIKE, () => {
+      NotificationManager.info(NotificationMessage.LIKED_POST);
     });
-    socket.on('new_post', post => {
+    socket.on(SocketEvent.NEW_POST, post => {
       if (post.userId !== id) {
         onPostApply(post.id);
       }
     });
 
     return () => {
+      socket.emit(SocketEvent.LEAVE_ROOM, id);
       socket.close();
```

Comment:
use socket.removeAllListeners() instead of close because after relogin it won't reconnect

Reply:
Good catch 💫